AATAMS/AATAMS_acoustic/check_csv.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
+"""
+Check that csv file has required fields 
+"""
+import os
+import sys
+import pandas
+
+csvfile = sys.argv[1]
+header_col = ['transmitter_id', 'installation_name', 'station_name', 'receiver_name', 'detection_timestamp', 'longitude', 'latitude',
+              'sensor_value', 'sensor_unit', 'FDA_QC', 'Velocity_QC', 'Distance_QC', 'DetectionDistribution_QC', 'DistanceRelease_QC',
+              'ReleaseDate_QC', 'ReleaseLocation_QC', 'Detection_QC']
+
+# CSV file to extract array of formatted data
+data = pandas.read_csv(csvfile, delimiter=';', header=0)
+
+if set(data.columns.values).issubset(header_col) == False:
```

Comment:
I am testing that the statement  -every columns of the data is in the header_col - is false . I agree it is overcomplicated but it is right.
Anyway i'll adopt your suggested command